Unix/tests/codec/mof/blue/qualifierset.c

```diff
@@ -18,6 +18,12 @@
 # define STRCASECMP _wcsicmp
 #else
 #include <wchar.h>
+
+#if !defined(CONFIG_HAVE_WCSCASECMP)
+// defined in pal/strings
+int wcscasecmp(const wchar_t* s1, const wchar_t* s2);
+#endif
```

Comment:
See comment from parameterset.c. But I see here that you're including `wchar.h`, and yet you're still defining this. So, obviously, certain platforms don't have this. I see it can depend on POSIX version, which is likely what you're running into.

You can up the posix version on various UNIX platforms, which would likely avoid the need for this. But given that we want to get this committed, it's not worth it at this time. Or, in other words, don't worry about this if you really need it.